webpack/webpack.js

```diff
@@ -107,5 +116,9 @@ module.exports = env => ({
         }),
         new SitmapGeneratorPlugin(env),
         new webpack.ProgressPlugin(),
-    ]
+        new webpack.ProvidePlugin({
+            process: 'process',
+            Buffer: ['buffer', 'Buffer'],
```

Comment:
Again, these are imperative for `react-docgen` to be able to be imported correctly.

(*Took me hours to get this to work properly*)